### PR TITLE
camel case angular (kebab-case removal)

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

**Comment by vicb** [comment]
mention "kebab-case removal in the commit message"

File: modules/angular2/docs/migration/TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+Angular2 is now fully case sensitive.
```
Comment (tbosch) [inline modules/angular2/docs/migration/TEMPLATE.md]: lowercase filename

Comment (tbosch) [inline modules/angular2/docs/migration/TEMPLATE.md]: Migration from which version to which version, maybe in filename and in the file itself?

File: modules/angular2/docs/migration/TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  - bind to the `someEvent` event,
+  - bind to the `someProperty` property,
+  - create a `someVar` local variable
+
+You should now write:
+
+    <my-cmp (someEvent)="someAction()" [someProperty]="expression" #someVar>
+
+Notes:
+  - while tag name are case sensitives, the best practice is to use dash case for component elements so that the browser
```
Comment (tbosch) [inline modules/angular2/docs/migration/TEMPLATE.md]: sensitive

File: modules/angular2/docs/migration/TEMPLATE.md (hunk 1 above)
Comment (tbosch) [inline modules/angular2/docs/migration/TEMPLATE.md]: Remove `fully`

File: modules/angular2/docs/migration/TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+    @Directive({inputs: ['myProp', 'myOtherProp: my-attr-name']})
+
+As attribute names are now case sensitive, they should be converted from dash to camel case where they are specified.
+The previous decorator becomes:
+
+    @Directive({inputs: ['myProp', 'myOtherProp: myAttrName']})
+
+Notes:
+  - only the long syntax (with ":") need not be updated,
```
Comment (tbosch) [inline modules/angular2/docs/migration/TEMPLATE.md]: needs

File: modules/angular2/docs/migration/TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  - `properties` is the legacy name for `inputs` and should updated in the same way - it is a good idea to replace
```
Comment (tbosch) [inline modules/angular2/docs/migration/TEMPLATE.md]: should be

File: modules/angular2/docs/migration/TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+        '[someProp]': 'exp',
+        '[style.background-color]': 'exp',
+        '[class.some-class]': 'exp',
+        '(someEvent)': 'action()',
+        'some-attr': 'value'
+      }
+    })
+
+The property bindings (`[...]`) and event bindings (`(...)`) must be updated in the same way as they are updated in a 
+template (reminder: `[attr.]` and `[class.]` should not be converted to camel case).
```
Comment (tbosch) [inline modules/angular2/docs/migration/TEMPLATE.md]: Add `style`

File: modules/angular2/src/compiler/template_parser.ts [hunk 6]
```diff
@@ -356,7 +354,10 @@ class TemplateParseVisitor implements HtmlAstVisitor {
 
   private _parseVariable(identifier: string, value: string, sourceSpan: ParseSourceSpan,
                          targetVars: VariableAst[]) {
-    targetVars.push(new VariableAst(dashCaseToCamelCase(identifier), value, sourceSpan));
+    if (identifier.indexOf('-') > -1) {
+      this._reportError(`"-" not allowed in variable names`, sourceSpan);
```
Comment (tbosch) [inline modules/angular2/src/compiler/template_parser.ts]: is not

File: modules/angular2/src/compiler/template_parser.ts [hunk 7]
```diff
@@ -547,20 +545,20 @@ class TemplateParseVisitor implements HtmlAstVisitor {
             sourceSpan);
       }
     } else {
-      let lcPrefix = parts[0].toLowerCase();
-      if (lcPrefix == ATTRIBUTE_PREFIX) {
-        boundPropertyName = dashCaseToCamelCase(parts[1]);
+      if (parts[0].toLowerCase() != parts[0]) {
```
Comment (tbosch) [inline modules/angular2/src/compiler/template_parser.ts]: Remove this case

File: modules/angular2/test/compiler/selector_spec.ts [hunk 8]
```diff
@@ -338,10 +343,10 @@ export function main() {
 
   describe('CssSelector.getMatchingElementTemplate', () => {
     it('should create an element with a tagName, classes, and attributes', () => {
```
Comment (tbosch) [inline modules/angular2/test/compiler/selector_spec.ts]: add `with the right casing`

File: modules/angular2/test/upgrade/upgrade_spec.ts [hunk 9]
```diff
@@ -18,7 +18,8 @@ import * as angular from 'angular2/src/upgrade/angular_js';
 
 export function main() {
   if (!DOM.supportsDOMEvents()) return;
-  describe('adapter: ng1 to ng2', () => {
+  // TODO(vicb): fix the code & re-enable
```
Comment (tbosch) [inline modules/angular2/test/upgrade/upgrade_spec.ts]: Fix this TODO

File: modules/angular2/docs/migration/TEMPLATE.md (hunk 1 above)
Comment (tbosch) [inline modules/angular2/docs/migration/TEMPLATE.md]: Mention changing styles as well

**Comment by vicb** [comment]
closing this PR as Travis does not pick the updates to the branch - will re-open a fresh one

**Pull request closed without merging**